osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceCommunicationInformationService.java

```diff
@@ -8,15 +8,14 @@
  */
 package org.opensmartgridplatform.core.application.services;
 
-import org.opensmartgridplatform.domain.core.entities.Device;
+import org.joda.time.DateTime;
```

Comment:
Ik zou hier geen joda time gebruiken, maar java time, we hebben een tijdje geleden alle joda time vervangen door java time

Reply:
Heb het aangepast